spec/hydra/works/services/generic_file/add_related_object_spec.rb

```diff
@@ -47,12 +45,10 @@
 
         it 'should solrize member ids' do
           skip 'skipping this test because issue #109 needs to be addressed' do
-          expect(subject.to_solr["generic_files_ssim"]).to include(generic_file2.id,generic_file1.id)
-          expect(subject.to_solr["generic_files_ssim"]).not_to include(object1.id,object2.id,file1.id,file2.id)
           expect(subject.to_solr["files_ssim"]).to include(file1.id,file2.id)
-          expect(subject.to_solr["files_ssim"]).not_to include(object1.id,object2.id,generic_file1.id,generic_file2.id)
-          expect(subject.to_solr["related_objects_ssim"]).to include(object1.id,object2.id)
-          expect(subject.to_solr["related_objects_ssim"]).not_to include(generic_file2.id,generic_file1.id,file1.id,file2.id)
+          expect(subject.to_solr["files_ssim"]).not_to include(object1.id,object2.id,generic_file1)
```

Comment:
Should `generic_file1` on lines 49-50 be `generic_file1.id`?  The other `include` checks check identifiers explicitly.